site/documentation/content/user-guide/amqp-adapter.md

```diff
@@ -141,39 +140,45 @@ There are two JAR files in the hono/cli/target directory. The JAR to use for the
 
 All other combinations are not supported by the adapter and will result in the message being ignored (pre-settled) or rejected (unsettled).
 
 ## Publish Telemetry Data (authenticated Device)
 
-The AMQP adapter supports publishing of telemetry data to Hono's Telemetry API. Telemetry messages can be published using either *AT LEAST ONCE* or *AT MOST ONCE* delivery semantics.
-
-* Message Address: `telemetry` or `t`
-  * This refers to the `to` property of the message.
-* Settlement Mode: `presettled` (*AT MOST ONCE*) or `unsettled` (*AT LEAST ONCE*)
 * Authentication: SASL PLAIN or SASL EXTERNAL
+* Message *properties*:
+  * (required) `to`: either `telemetry` or `t`
+  * (optional) `content-type`: The type of payload contained in the message body. The given content type will be used
+    in the AMQP message being forwarded downstream if not empty. Otherwise, the content type of the downstream
+    message will be set to `application/octet-stream` if the payload is not empty and no default content type has been
+    defined for the origin device or its tenant (see [Downstream Meta Data]({{< relref "#downstream-meta-data" >}}).
 * Message Body:
-  * (optional) Arbitrary payload
-* Message properties:
-  * (optional) Arbitrary properties (content-type, correlation-id, ...)
-* Disposition Frames:
-  * Accepted: Message successfully processed by the adapter.
-  * Released: Message cannot be processed and should be redelivered.
-  * Rejected: Adapter rejects the message due to one of the following:
-        * (`hono:bad-request`): Request rejected due to a bad client request.
-        * (`amqp:unauthorized-access`): Request rejected because the adapter is disabled for tenant.
-        * (`amqp:precondition-failed`): Request does not fulfill certain requirements e.g adapter cannot assert device registration etc.
-        * (`amqp:resource-limit-exceeded`): Request rejected because the message limit for the given tenant is exceeded.
-
-When a device publishes data to the `telemetry` address, the AMQP adapter automatically determines the device's identity and tenant during the authentication process.
+  * (optional) Arbitrary payload contained in either a single AMQP *Data* or *AmqpValue* section.
+* Outcomes:
+  * `accepted`: The message has been successfully forwarded downstream.
+  * `released`: The message could not be processed by the adapter due to a (temporary) problem at the infrastructure level.
+    A common reason for this is that there is no active downstream consumer for messages from the device.
```

Comment:
I think "by the adapter" and "at the infrastructure level" are potentially misleading for the telemetry case here. The adapter could have forwarded it just fine, but the northbound application for example may not have sent a disposition update in time, leading to the (initially unsettled) message to get released.

Reply:
From the client's view, it is the adapter which processes its uploaded message. The client does not know (nor need to know) if and how the adapter is forwarding the message. However, the message will also be released if one of the registry services cannot be reached by the adapter and therefore I think that it is appropriate to refer to *infrastructure* level in general here.

Reply:
Then I would at least say "potentially at the infrastructure level".
A developer responsible both for the device and the northbound application side could otherwise mistakenly read this error as "not my fault" while perhaps having forgotten to do a delivery update in the northbound application.